opal/mca/patcher/base/patcher_base_patch.c

```diff
@@ -84,7 +85,7 @@ static int PatchLoadImm(uintptr_t addr, unsigned int reg, size_t value)
 
 #endif
 
-#if !HAVE___CLEAR_CACHE
```

Comment:
This define is never really defined. This code might need a more careful validation.